pycds/orm/tables.py

```diff
@@ -607,3 +609,158 @@ class DerivedValue(Base):
+    # - String types do not provide lengths. All strings are ultimately variable
+    #   length in PG and there seems no reason to limit them. (Except perhaps to
+    #   prevent entry of erroneous values that just happen to be very long?)
+    #
+    # - None are nullable. In contrast, most in the model tables are.
+    __tablename__ = "climo_station"
+    id = Column("climo_station_id", Integer, primary_key=True)
+    type = Column(
+        Enum("long-record", "composite", "prism", name="climo_station_type_enum"),
+        nullable=False,
+    )
+    basin_id = Column(Integer, nullable=True)
+    comments = Column(String, nullable=False)
+
+
+class ClimatologicalStationHistory(Base):
+    __tablename__ = hx_table_name(ClimatologicalStation.__tablename__, schema=None)
+    climo_station_id = Column(Integer, nullable=False, index=True)
+    type = Column(
+        Enum("long-record", "composite", "prism", name="climo_station_type_enum"),
+        nullable=False,
+    )
+    basin_id = Column(Integer, nullable=True)
+    comments = Column(String, nullable=False)
+    mod_time = Column(DateTime, nullable=False, server_default=func.now())
+    mod_user = Column(
+        String(64), nullable=False, server_default=literal_column("current_user")
+    )
+    deleted = Column(Boolean, default=False)
+    climo_station_hx_id = Column(Integer, primary_key=True)
+    climo_period_hx_id = Column(
+        Integer, ForeignKey("climo_period_hx.climo_period_hx_id")
+    )
+
+
+class ClimatologicalStationXHistory(Base):
+    __tablename__ = "climo_stn_x_hist"
+    climo_station_id = Column(
+        Integer,
+        ForeignKey("climo_station.climo_station_id"),
+        primary_key=True,
+    )
+    history_id = Column(
+        Integer,
+        ForeignKey("meta_history.history_id"),
+        primary_key=True,
+    )
+    role = Column(Enum("base", "joint", name="climo_station_role_enum"), nullable=False)
+
+
+class ClimatologicalVariable(Base):
+    # TODO: Columns in this table parallel those in meta_station and meta_history.
+    # They differ in the following ways, which may be questioned:
+    #
+    # - String types do not provide lengths. All strings are ultimately variable
+    #   length in PG and there seems no reason to limit them. (Except perhaps to
+    #   prevent entry of erroneous values that just happen to be very long?)
+    #
+    # - None are nullable. In contrast, most in the model tables are.
+    __tablename__ = "climo_variable"
+
+    id = Column("climo_variable_id", Integer, primary_key=True)
+    duration = mapped_column(
+        Enum("annual", "seasonal", "monthly", name="climatology_duration_enum"),
+        nullable=False,
+    )
+    unit = Column(String, nullable=False)
+    standard_name = Column(String, nullable=False)
+    display_name = Column(String, nullable=False)
+    short_name = Column(String, nullable=False)
+    cell_methods = Column(String, nullable=False)
+    net_var_name = Column(CIText(), nullable=True)
```

Comment:
Suggest we check with scientist(s) and/or James which of these columns are useful.